index.bs

```diff
@@ -5116,6 +5121,44 @@ aborted</dfn> steps given |navigable| and |navigation status|:
 
 </div>
 
+#### The browsingContext.navigationCommitted Event #### {#event-browsingContext-navigationCommitted}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        browsingContext.NavigationCommitted = (
+         method: "browsingContext.navigationCommitted",
+         params: browsingContext.NavigationInfo
+        )
+      </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The [=remote end event trigger=] is the <dfn export>WebDriver BiDi navigation committed</dfn> steps
+given |navigable| and |navigation status|:
+
+1. Let |params| be the result of [=get the navigation info=] given |navigable|
+   and |navigation status|.
+
+1. Let |body| be a [=/map=] matching the
+    <code>browsingContext.NavigationCommitted</code> production, with the
+    <code>params</code> field set to |params|.
+
+1. Let |navigation id| be |navigation status|'s id.
+
+1. Let |related navigables| be a [=/set=] containing |navigable|.
+
```

Comment:
Lets use the same order as the other navigation events:

```suggestion
1. Let |related navigables| be a [=/set=] containing |navigable|.

1. Let |navigation id| be |navigation status|'s id.

```

Reply:
Well, the current order is the same as the order of steps in navigationFailed.

Reply:
But different to the `domContentLoaded` and `load`. Would be nice to have it in the same order everywhere for navigation events.

Reply:
So I am not sure the order matters for anything but I'd suggest to open a follow-up editorial PR to update those if it is important for anything. I applied your suggestion nonetheless.